### ISSUE TITLE
Error publishing app to the app store

### ISSUE THREAD

**Issue by erjdriver** [Notalib/nativescript-webview-ext#71]
My app builds on my local pc.

When I try to publish it to the app store (ios version) - I get:

```
ERROR ITMS-90206: "Invalid Bundle. The bundle at 'publishmyappios.app/Frameworks/NotaWebViewExt.framework' contains disallowed file 'Frameworks'."
[2020-04-18 17:03:24 EDT] <main> DBG-X: Returning 1
Command /Applications/Xcode.app/Contents/SharedFrameworks/ContentDeliveryServices.framework/Versions/A/itms/bin/iTMSTransporter failed with exit code 1
```

I looked up the `disallowed file frameworks` message and it seems to do with how the plugin/extension is built.

https://stackoverflow.com/questions/25777958/validation-error-invalid-bundle-the-bundle-at-contains-disallowed-file-fr?lq=1

Thank you.

p.s. this error is during the publish stage - buiids ok.

**Comment by m-abs** [comment]
Hi,

I don't have a way to test this at the moment, but could you try something for me?

Delete the folders:
```bash
node_modules/@nota/nativescript-webview-ext/platforms/ios/NotaWebViewExt.xcframework/ios-armv7_arm64/NotaWebViewExt.framework/Frameworks
node_modules/@nota/nativescript-webview-ext/platforms/ios/NotaWebViewExt.xcframework/ios-i386_x86_64-simulator/NotaWebViewExt.framework/Frameworks
```

When rebuild?

**Comment by erjdriver** [comment]
That worked - great!

**Comment by m-abs** [comment]
Thanks for reporting and testing.

This is a real bug.
I'm reopening the issue. It should be fixes once v6.5.2 is released.

**Comment by m-abs** [comment]
Hi @erjdriver 

I've just published v6.5.2.
